modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java

```diff
@@ -3184,7 +3290,7 @@ public int execute(List<String> rawArgs) {
                             break;
 
                         case CACHE:
-                            cache(client, args.cacheArgs());
+                            lastOperationRes = cache(client, args.cacheArgs());
```

Comment:
You have changed method signature, but behavior (lastOperationRes now shared not as side effect, but as return value) is changed for this call only. Such difference may raise questions in code reader's head. Shall we change behavior for another calls as well?

Reply:
Would do it in another ticket.